host/xtest/regression_4000_data.h

```diff
@@ -8486,4 +8486,270 @@ static const uint8_t mac_cmac_vect12_out[] = {
+	0x6b, 0xc1, 0xbe, 0xe2, 0x2e, 0x40, 0x9f, 0x96
+};
+
+/*
+ * btw, test vector for the Example 14 is not correct in the forementioned document
+ * it claims that the result should be equal to b7a688e1 22ffaf95 but in reality
+ * it should be equal to 8e8f2931 36283797 instead
+ * Verified with Nettle and mbedTLS
+ */
+/* Example 14 */
+static const uint8_t mac_des3_cmac_vect2_out[] = {
+	0x8e, 0x8f, 0x29, 0x31, 0x36, 0x28, 0x37, 0x97
+};
+
+#define mac_des3_cmac_vect3_key mac_des3_cmac_vect1_key
+static const uint8_t mac_des3_cmac_vect3_data[] = {
+	0x6b, 0xc1, 0xbe, 0xe2, 0x2e, 0x40, 0x9f, 0x96,
+	0xe9, 0x3d, 0x7e, 0x11, 0x73, 0x93, 0x17, 0x2a,
+	0xae, 0x2d, 0x8a, 0x57
+};
+
+/*
+ * btw, test vector for the Example 15 is not correct in the forementioned document
+ * it claims that the result should be equal to d32bcebe 43d23d80 but in reality
+ * it should be equal to 743ddbe0 ce2dc2ed instead
+ * Verified with Nettle and mbedTLS
+ */
+
+/* Example 15 */
+static const uint8_t mac_des3_cmac_vect3_out[] = {
+	0x74, 0x3d, 0xdb, 0xe0, 0xce, 0x2d, 0xc2, 0xed
+};
+
+#define mac_des3_cmac_vect4_key mac_des3_cmac_vect1_key
+static const uint8_t mac_des3_cmac_vect4_data[] = {
+	0x6b, 0xc1, 0xbe, 0xe2, 0x2e, 0x40, 0x9f, 0x96,
+	0xe9, 0x3d, 0x7e, 0x11, 0x73, 0x93, 0x17, 0x2a,
+	0xae, 0x2d, 0x8a, 0x57, 0x1e, 0x03, 0xac, 0x9c,
+	0x9e, 0xb7, 0x6f, 0xac, 0x45, 0xaf, 0x8e, 0x51
+};
+
+/* Example 16 */
+static const uint8_t mac_des3_cmac_vect4_out[] = {
+	0x33, 0xe6, 0xb1, 0x09, 0x24, 0x00, 0xea, 0xe5
+};
+
+/* Two Key TDEA */
+static const uint8_t mac_des3_cmac_vect5_key[] = {
+	0x4c, 0xf1, 0x51, 0x34, 0xa2, 0x85, 0x0d, 0xd5,
+	0x8a, 0x3d, 0x10, 0xba, 0x80, 0x57, 0x0d, 0x38,
+	0x4c, 0xf1, 0x51, 0x34, 0xa2, 0x85, 0x0d, 0xd5
+};
+
+/* Example 17 */
+static const uint8_t mac_des3_cmac_vect5_out[] = {
+	0xbd, 0x2e, 0xbf, 0x9a, 0x3b, 0xa0, 0x03, 0x61
+};
+
+
+#define mac_des3_cmac_vect6_key mac_des3_cmac_vect5_key
+static const uint8_t mac_des3_cmac_vect6_data[] = {
+	0x6b, 0xc1, 0xbe, 0xe2, 0x2e, 0x40, 0x9f, 0x96
+};
+
+/* Example 18 */
+/*
+ * btw, test vector for the Example 18 is not correct in the forementioned document
+ * it claims that the result should be equal to bd2ebf9a 3ba00361 but in reality
+ * it should be equal to 4ff2ab81 3c53ce83 instead
+ * Verified with Nettle and mbedTLS
+ */
+static const uint8_t mac_des3_cmac_vect6_out[] = {
+	0x4f, 0xf2, 0xab, 0x81, 0x3c, 0x53, 0xce, 0x83
+};
+
+#define mac_des3_cmac_vect7_key mac_des3_cmac_vect5_key
+static const uint8_t mac_des3_cmac_vect7_data[] = {
+	0x6b, 0xc1, 0xbe, 0xe2, 0x2e, 0x40, 0x9f, 0x96,
+	0xe9, 0x3d, 0x7e, 0x11, 0x73, 0x93, 0x17, 0x2a,
+	0xae, 0x2d, 0x8a, 0x57
+};
+
+/* Example 19 */
+/*
+ * btw, test vector for the Example 19 is not correct in the forementioned document
+ * it claims that the result should be equal to 8ea92435 b52660e0 but in reality
+ * it should be equal to 62dd1b47 1902bd4e instead
+ * Verified with Nettle and mbedTLS
+ */
```

Comment:
That's a lot of mistakes in this NIST publication :-/ a good reason to use the more recent one you mentioned above IMO.

Reply:
I'm using both in this PR, just in case:)
Do you want me to remove the ones from the obsolete document along with my comments (btw, ...)?

Reply:
If coverage is enough without the "obsolete" ones, then yes I'd rather not have them.